Source/Operations/Operation.cs

```diff
@@ -90,6 +106,11 @@ protected virtual MergeResult DoMerge(Operation operation)
 			return MergeResult.Continue;
 		}
 
+		protected string ToIndentedQuery()

```

Comment:
I might structure as `AsIndentedQuery(ToQuery());` to minimize hidden virtual method call.